### PR TITLE
fixed local run of integration tests

### PR DESCRIPTION
**Pull request by alexvas**
## Changes

- Removed broken CA/client certificates with client private key from repository. 
- Fixed keypairs generating script, added two another scripts to install generated CA script in Ubuntu
- Updated documentation

## Verification
```
yarn test:integration
```
works for me now. Let's see if travis integration would work with new setup process.

### CODE REVIEW

**Comment by johanbrandhorst** [comment]
Seems like you need to source the bash script before generating the key pair.

File: integration_test/go/testserver/testserver.go [hunk 1]
```diff
@@ -88,7 +88,7 @@ func main() {
 		Handler: http.HandlerFunc(handler),
 	}
 	http2EmptyServer := http.Server{
-		Addr:    fmt.Sprintf(":%d", *http2EmptyPort),
+		Addr:    fmt.Sprintf("localhost:%d", *http2EmptyPort),
```
[the remark this answers is not in the document]
Comment (alexvas) [reply integration_test/go/testserver/testserver.go]: done:
https://github.com/improbable-eng/grpc-web/pull/585

**Comment by alexvas** [comment]
> I feel a bit weird about adding temporary certificates into my system trust store

As long as it is the same `/etc/ssl/certs/grpc_web_localhostCA.pem` it does not bother me personally.

**Comment by johanbrandhorst** [comment]
Closing in favour of #585

**Pull request closed without merging**